Lib/importlib/_bootstrap_external.py

```diff
@@ -1162,8 +1162,8 @@ class ExtensionFileLoader(FileLoader, _LoaderBasics):
 
     """
 
-    def __init__(self, name, path):
-        self.name = name
+    def __init__(self, fullname, path):
```

Comment:
The idea is to make `name` deprecated first and then switch to only allowing `fullname`.